generator/TestServiceModels/restjson-tests-client/rest-json-protocol-2019-12-16.normal.json

```diff
@@ -389,6 +389,17 @@
       "output":{"shape":"JsonMapsInputOutput"},
       "documentation":"<p>The example tests basic map serialization.</p>"
     },
+    "SparseJsonMaps":{
```

Comment:
These files come from a repo and can't be changed. If you did this to get the models generated that is fine but you will have to copy the generated code to the partial class and then undo these *.json files.